manifests/e/EUSO/eParakstītājs3/0/1.7.2/EUSO.eParakstītājs3.0.locale.LV.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: EUSO.eParakstītājs3.0
+PackageVersion: 1.7.2
+PackageLocale: LV
+Publisher: EUSO
+PackageName: eParakstītājs 3.0
+License: eParaksts  2017 VAS "Latvijas Valsts radio un televīzijas centrs"

```

Comment:
An eSignature is not a license. Please update to actual licence terms such as `Proprietary`, `MIT`, `GPL-3.0`, or similar